### DISCUSSION TITLE
TypeError: can't access property "doc", a is undefined

### DISCUSSION THREAD

**Discussion by dzhuravkov** [richardeschloss/nuxt-highcharts#70] [Q&A]
Hello, I get this error (TypeError: can't access property "doc", a is undefined) when I am trying to use the highstock component. If I use highchart or highmap I don't have an issue. Also, I tried to use ClientOnly and v-if state with a button that can show/hide the chart.

my package.json main info:
```
...
"nuxt": "^3.6.5",
"typescript": "^4.9.5",
"nuxt-highcharts": "^3.0.11",
"highcharts": "9.3.2",
...
```

component:
```
<template>
  <highstock />
</template>
```

nuxt.config:
```
...
  modules: [
    '@vueuse/nuxt',
    '@nuxtjs/google-fonts',
    'nuxt-highcharts'
  ],
 ...
```
![image](https://github.com/richardeschloss/nuxt-highcharts/assets/98736012/11c89a55-47e9-4283-8f38-ac444723a048)

Maybe you have an idea of how I can solve this issue. Thank you!

**Comment by richardeschloss** [comment]
I'm not sure, perhaps chart options need to be defined and passed to the highstock component? 
Here's a working StockChart example: https://github.com/richardeschloss/nuxt-highcharts/blob/master/components/charts/StockChart.vue . How far does this example get you?
